### PR TITLE
Address accessibility issues of Mastodon rel=me link

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/open-telemetry/opentelemetry.io/issues/2199

### CODE REVIEW

**Review by chalin — CHANGES_REQUESTED** [review]
As I mentioned in the issue, this link isn't visible and so this addition is unnecessary.

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {{ partial "head.html" . }}
   </head>
   <body class="td-{{ .Kind }}{{ with .Page.Params.body_class }} {{ . }}{{ end }}">
-    <a rel="me" href="https://fosstodon.org/@opentelemetry"></a>
+    <a rel="me" href="https://fosstodon.org/@opentelemetry" aria-label="fosstodon.org/@opentelemetry"></a>
```
Comment (chalin) [inline layouts/_default/baseof.html:9]: If we must add an ARIA attribut, then this element should be declared as [hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden):

```suggestion
    <a rel="me" href="https://fosstodon.org/@opentelemetry" aria-hidden="true"></a>
```

Comment (Arhell) [reply layouts/_default/baseof.html]: Done thanks

**Review by chalin — APPROVED** [review]
Ah yes, we do need to deal with tabbing too. Thanks!

Now that I've had a bit of time to look into this more, I see that further changes are needed. I'll deal with them while working on #2104.

**Pull request merged**